ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-main.yaml

```diff
@@ -3,11 +3,16 @@ base_images:
     name: "4.10"
     namespace: ocp
     tag: base
+  ocp_builder_rhel-8-golang-1.18-openshift-4.11:
```

Comment:
OCP 1.4.0 will be build against rhel9, golang 1.19 and OCP 4.13.  Prow should be consistent with our builds.